hack/make/.build-rpm/docker-engine.spec

```diff
@@ -35,6 +35,7 @@ Requires(preun): initscripts
 
 # required packages on install
 Requires: /bin/sh
+Requires: docker-selinux
```

Comment:
this installs their docker-selinux pacakge.... do we trust that it has broken our stuff in the past.